src/tree/ActionTreeItem.ts

```diff
@@ -7,15 +7,15 @@ import { Octokit } from '@octokit/rest';
 import { ActionsGetWorkflowRunResponseData, ActionsListJobsForWorkflowRunResponseData, OctokitResponse } from '@octokit/types';
 import { AzExtTreeItem, AzureParentTreeItem, IActionContext, TreeItemIconPath } from "vscode-azureextensionui";
 import { createOctokitClient } from '../commands/github/createOctokitClient';
-import { getActionIconPath } from '../utils/actionUtils';
+import { ensureStatus, getActionIconPath } from '../utils/actionUtils';
 import { getRepoFullname } from '../utils/gitHubUtils';
 import { ActionsTreeItem } from "./ActionsTreeItem";
 import { IAzureResourceTreeItem } from './IAzureResourceTreeItem';
 import { JobTreeItem } from './JobTreeItem';
 
 export class ActionTreeItem extends AzureParentTreeItem implements IAzureResourceTreeItem {
-    public static contextValue: string = 'azureStaticAction';
-    public readonly contextValue: string = ActionTreeItem.contextValue;
+    public static contextValueCompleted: string = 'azureStaticActionCompleted';
+    public static contextValueInProgress: string = 'azureStaticActionInProgress';
```

Comment:
> I don't think there's a great way to fix this through the command palette since pickTreeItemImpl can't return an array of children (which I could filter).

I think you need `isAncestorOfImpl` instead of `pickTreeItemImpl`. Something as simple as this:
```typescript
return contextValue !== ActionTreeItem.contextValueCompleted && contextValue !== ActionTreeItem.contextValueInProgress;
```

That basically just tells the tree item picker that this is the last level of the tree you can find those contextValues.

Reply:
While you're fixing the command palette experience, might as well add a better "noItemFoundErrorMessage", too